.changeset/eighty-geckos-rush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/auth': patch
+---
+
+minor refactors to router events on InitPage, minor changes to SignupContainer styles
```

Comment:
Because changeset text ends up in changelogs we write it in past tense and full sentences (capital letters, full stops).

```suggestion
Refactored router events on InitPage, and made minor changes to SignupContainer styles.
```

Reply:
argh i keep forgetting this, thanks Tim